lib/galaxy/datatypes/molecules.py

```diff
@@ -62,6 +68,121 @@ def set_peek(self, dataset):
     def get_mime(self):
         return 'text/plain'
 
+    def get_element_symbols(self):
```

Comment:
I suppose this could be a `element_symbols` class attribute instead of a method?

Reply:
Sure, done.